README.md

```diff
@@ -187,6 +187,37 @@ result = my_zosmf.get_info()
 ```
 The result will be a JSON object containing z/OSMF information
 
+## Local project setup (Windows)
```

Comment:
Does this section only apply to Windows? Linux or macOS users could technically follow the same steps without issue.